docs/UserGuide.md

```diff
@@ -160,33 +289,94 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+AddressBook data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Replace the term  "AddressBook" with "SociaLite".

Reply:
Oh right i missed that out! Thanks!